### ISSUE TITLE
Can't get the pro features to work

### ISSUE THREAD

**Issue by 0x80** [wallabyjs/quokka#30]
### Issue description or question

I can't get the pro features to work (performance test, show expression result). I set {"pro":true} in my ~/.quokka/config.json and also reinstalled Quokka so that it could generate a new config. 

### Quokka.js file

```javascript
const add = (a, b) => a + b
const result = add(2345, 5467) /*?*/
console.log(result)
```

### Quokka.js Console Output

```
​​​​Quokka #2 (node: v7.6.0, babel: v6.23.1)​​​​

​​​​​7812​​​​​ at ​​​result​​​ ​sketches/quokka.js:5:0​

```

### Code editor version

Visual Studio Code v1.10.2

### OS name and version

OSX 10.12.3

**Comment by ArtemGovorov** [comment]
With babel you need to add `;` to make it work:

```javascript
const add = (a, b) => a + b
const result = add(2345, 5467) /*?*/; // <---
console.log(result)
```

Without babel it'll work as is, but babel insert `;` before the comment if it's missing. To see what I mean compare [this](https://babeljs.io/repl/#?babili=false&evaluate=true&lineWrap=false&presets=es2015&targets=&browsers=&builtIns=false&experimental=true&loose=false&spec=false&code=const%20add%20%3D%20(a%2C%20b)%20%3D%3E%20a%20%2B%20b%0Aconst%20result%20%3D%20add(2345%2C%205467)%20%2F*%3F*%2F%0Aconsole.log(result)&playground=true&stage=0) and [this](https://babeljs.io/repl/#?babili=false&evaluate=true&lineWrap=false&presets=es2015&targets=&browsers=&builtIns=false&experimental=true&loose=false&spec=false&code=const%20add%20%3D%20(a%2C%20b)%20%3D%3E%20a%20%2B%20b%0Aconst%20result%20%3D%20add(2345%2C%205467)%20%2F*%3F*%2F%3B%0Aconsole.log(result)&playground=true&stage=0).

**Comment by ArtemGovorov** [comment]
I have also added the [note about it to our docs](https://quokkajs.com/docs/index.html#comments).

**Comment by 0x80** [comment]
Thanks. That is a little bit unfortunate. I'm using prettier with prettier-eslint to format code automatically according to eslint rules. I am super happy with it since it seems solid enough to actually auto-format my code on save.

In order to work around my eslint rules and have the formatter not strip the comments like that I had to fiddle with it a bit, but I'm happy I got it to work:

```javascript
/*eslint-disable semi, no-inline-comments, semi-spacing*/
/*eslint spaced-comment: ["error", "never", { "block": { "balanced": true } }]*/

const add = (a, b) => a + b
const result = add(2345, 5467) /*?*/; // <---
console.log(result)
```

Of course it's not ideal, but I'm using Quokka only in separate sketch files so I guess this is an ok workaround.

**Comment by 0x80** [comment]
Seems like I have a problem after all 😞 The eslint rule for semi-spacing either removes the comment, or it forces a space before the comment. In the example above the space prevents Quokka from showing the result value.

Would you consider fixing this in Quokka to make it less strict? I have no idea how to work around this now.

**Comment by ArtemGovorov** [comment]
We have just released the new 1.0.24 version, where we now support cases like:

```javascript
const result = add(2345, 5467) /*?*/
const result = add(2345, 5467); /*?*/
const result = add(2345, 5467) /*?*/;
const result = add(2345, 5467)/*?*/;
```
(with or without babel)

<img width="411" alt="screen shot 2017-04-06 at 12 48 51 pm" src="https://cloud.githubusercontent.com/assets/979966/24735756/cfbf56fe-1ac7-11e7-88f9-f3639f4617c8.png">

**Comment by 0x80** [comment]
That was fast! 👍 

It would be completely perfect if you could make it so that `/* ? */` is also recognised,  because then Quokka doesn't get in the way for people who like to keep their comments spaced (like with [standard](https://github.com/feross/standard)). 

I still have to disable comment spacing or make exceptions for strings like `?` and `?.` but that quickly gets out of hand:

`/* eslint spaced-comment: ["error", "always", { "block": { "balanced": true, "exceptions": ["?.", "?"]}}] */`

**Comment by ArtemGovorov** [comment]
Thanks for the note! 

> It would be completely perfect if you could make it so that `/* ? */` is also recognised

Done and published in v1.0.26:

![screen shot 2017-04-06 at 7 58 10 pm](https://cloud.githubusercontent.com/assets/979966/24748608/8b79deba-1b03-11e7-845d-c9b5adcc7df4.png)
